### DISCUSSION TITLE
add qoi as openexr compression

### DISCUSSION THREAD

**Discussion by octanejohn** [godotengine/godot-proposals#10518] [Engine Core]
like how qoa is being used in godot for wav

alternative
https://github.com/godotengine/godot-proposals/issues/3726

releated
https://github.com/godotengine/godot-proposals/discussions/8940

**Comment by Calinou** [comment]
QOI does not have support for HDR formats according to its [specification](https://qoiformat.org/qoi-specification.pdf). It can only store RGB8 or RGBA8.

**Reply by octanejohn** [reply]
ye i just saw that,  maybe when qoi2 comes out

releated
https://github.com/phoboslab/qoi/issues/2
https://github.com/nigeltao/qoi2-bikeshed/issues/4
